visualization.milkdrop2/resources/language/resource.language.af_za/strings.po

```diff
@@ -23,11 +23,3 @@ msgstr "MilkDrop laat jou sweef deur die klankgolwe wat jy hoor"
 msgctxt "Addon Description"
 msgid "MilkDrop was originally a music visualiser \"plug-in\" to the Winamp music player. As you listen to your music, MilkDrop takes you flying through the actual soundwaves you're hearing, and uses beat detection to trigger myriad psychedelic effects, creating a rich visual journey through sound."
 msgstr "MilkDrop was oorspronklik 'n musiek visualisering \"plug-in\" vir die Winamp musiek speler. Soos jy na jou musiek luister, laat MilkDrop jou deur die klankgolwe sweef wat jy hoor, en gebruik klop waarnemings om 'n magdom psigedeliese effekte te aktiveer, en 'n visuele reis deur klank te skep."
-
-#[Consistency] Make terms, settings names, brands and other minutiae, consistent throughout file.
-#[Capitalization] Avoid capitalizing every second word. See http://grammarist.com/capitalization/
-#For example, prefer wording as "This new string" instead of "This New String".
-#[Referencing] If a suitable string already exists, reuse it, making a note of where it's used!
-#[Description / location] For example, "#. Description of some setting" used on "#: xbmc/addons/guidialogaddoninfo.cpp"
-#When writing a description or setting, refer to a setting name in quotes. See existing entries for guidance.
-#For example, "Press \"OK\" for \"All seasons\"" instead of "Press OK for All seasons" after first word.
```

Comment:
@AlwinEsch Is it correct of Weblate to remove these lines?

Reply:
Not really, specially as it bring in on new files, no idea whats there happen.

If them away is also not problematic, was come in by me with c&p from another addon.

Reply:
Yeah, I saw that Weblate removes the lines in existing files and adds the exact same lines in new files.
What should we do?

Reply:
Think I create a request where removes them on all files and thing on next update here it goes away complete

Reply:
Ah, great.
I forced a refresh on a Weblate addon and that seems to remove the lines from all languages now.

Reply:
It force pushed to this PR.

Reply:
Thanks a lot! Then we use this and I change my other request to Nexus and takes the update commits from here.